content/millennium/r4/diagnostic/questionnaire-response.md

```diff
@@ -11,11 +11,25 @@ title: QuestionnaireResponse | R4 API
 
 ## Overview
 
-The QuestionnaireResponse resource is a collection of answers to a given questionnaire typically used to request patient healthcare information. An example of a questionnaire is a form used to collect a patient's social history. Currently, only social history is supported.
+The QuestionnaireResponse resource is a collection of answers to a given questionnaire typically used to request patient healthcare information. Only answer value types of String, Quantity, and Coding are supported. An example of a questionnaire is a form used to collect a patient's social history. Currently, only social history, which is a type of Observation, is supported. To find the corresponding social history questionnaire, first search QuestionnaireResponse by patient. As long as the patient exists, a QuestionnaireReponse will be returned even when the data does not exist. Take the 'questionnaire' field from the QuestionnaireResponse and use that to retrieve the original Questionnaire.
 
 The following fields are returned if valued:
 
 * [Id](https://hl7.org/fhir/R4/resource-definitions.html#Resource.id){:target="_blank"}
+* [Questionnaire](http://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.questionnaire){:target="_blank"}
+* [Status](http://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.status){:target="_blank"}
+* [Subject](http://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.subject){:target="_blank"}
+* [Encounter](http://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.encounter){:target="_blank"}
+* [Authored](http://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.authored){:target="_blank"}
+* [Item](http://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.item){:target="_blank"}
+    * [Link id](http://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.item.linkId){:target="_blank"}
+    * [Text](http://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.item.text){:target="_blank"}
+    * [Item](http://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.item.item){:target="_blank"}
+        * [Link id](http://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.item.linkId){:target="_blank"}
+        * [Text](http://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.item.text){:target="_blank"}
+        * [valueString Answers](http://hl7.org/fhir/r4/datatypes.html#string){:target="_blank"}
+        * [valueCoding Answers](http://hl7.org/fhir/r4/datatypes.html#Coding){:target="_blank"}
+        * [valueQuantity Answers](http://hl7.org/fhir/r4/datatypes.html#Quantity){:target="_blank"}
 
 ## Search
```

Comment:
need to documentation new search param 'patinet'

Reply:
added: 2516bdccd3b10f0071eab6c07aacc7110d0cba4f